SingularityUI/app/actions/api/slaves.es6

```diff
@@ -2,14 +2,14 @@ import { buildApiAction, buildJsonApiAction } from './base';
 
 export const FetchSlaves = buildApiAction(
   'FETCH_SLAVES',
-  {url: '/slaves'}
```

Comment:
the `buildApiAction` already has `/api/` so it's just an extra `/`. `../api//slaves` will work, but it's not proper so I figured i'd remove it. I can drop it from the rest of the files too, but prob better for another pr